test/util/memwallet.js

```diff
@@ -1295,6 +1295,10 @@ class MemWallet {
   async createUpdate(name, resource, options) {
     assert(typeof name === 'string');
 
+    if (resource === null) {
+      resource = Resource.fromJSON({records:[]});
+    }
```

Comment:
nit: you could remove `{}` for consistency with the code.

Reply:
Ok I hear ya. Will fix but I have gotten into this habit after reading: https://blog.codecentric.de/en/2014/02/curly-braces/